Misc/NEWS

```diff
@@ -10,6 +10,10 @@ What's New in Python 3.7.0 alpha 1?
 Core and Builtins
 -----------------
 
+- bpo-30876: Relative import from unloaded package now reimports the package
```

Comment:
I think the problem is because Larry add the pre-requirement on PyPI in 1.0.1 version: `Requires Python: >=3.6`. You can saw it at [1.0.1](https://pypi.python.org/pypi/blurb/1.0.1), since the latest version which didn't require Python 3.6 is 1.0, you will only get 1.0 on Python 3.5 :(